bin/load_genotyping_plates.pl

```diff
@@ -319,29 +309,43 @@ =head2 AUTHORS
 
 my $coderef= sub  {
 
+    my $trial = CXGN::Trial->new( { bcs_schema => $schema, trial_id => $genotyping_project_id });
+    my $location_data = $trial->get_location();
+    my $location_id = $location_data->[0];
```

Comment:
$location_id can be removed (using only $location_name)

Reply:
DONE.